.vscode/settings.json

```diff
@@ -102,11 +102,11 @@
   "mochaExplorer.require": "source-map-support/register",
   "mochaExplorer.debuggerConfig": "Tests",
   "mochaExplorer.env": {
-    "CADL_VERBOSE_TEST_OUTPUT": "true",
+    "TYPESPEC_VERBOSE_TEST_OUTPUT": "true",
     "NODE_OPTIONS": "--stack-trace-limit=50"
   },
   "prettier.prettierPath": "./packages/compiler/node_modules/prettier",
-  "prettier.documentSelectors": ["**/*.cadl"],
+  "prettier.documentSelectors": ["**/*.tsp"],
   "testExplorer.errorDecoration": false,
-  "cadl.cadl-server.path": "${workspaceFolder}/packages/compiler"
+  "typespec.tsp-server.path": "${workspaceFolder}/packages/compiler"
```

Comment:
We will need to document / handle in migrate tool these setting path changes.

Reply:
Good call. Would it be worth supporting both paths for a while?

Reply:
Will set aside .vscode for manual fix.

Reply:
Let's document this. 'cadl-> typesec' section